<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RawLabClass</class>
 <widget class="QMainWindow" name="RawLabClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RawLab</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#808080;
}

QMenu{
background-color:#A0A0A0;
color: #F0F0F0;
border: 1px solid #606060;
}
QMenu:Item{
background-color:#A0A0A0;
color: #F0F0F0;
}
QMenu:Item:selected{
background-color:#C0C0C0;
color: #404040;
}
QMenuBar{
background-color:#808080;
color: #E0E0E0;
}
QMenuBar:Item{
background-color:#808080;
color: #E0E0E0;
}
QMenuBar:Item:selected{
background-color:#A0A0A0;
color: #F0F0F0;
}

QToolBar{
border: 1px solid #606060;
}
QToolBar::separator:vertical{
border: 1px solid #A0A0A0;
width: 0px;
}
QToolButton:hover{
background-color: #A0A0A0;
border: none;
}

QToolTip {
background-color: #C0C0C0;
border: 1px solid #606060;
color: #404040;
padding: 0;
}

QStatusBar::item {
border: 1px solid #606060;
border-radius: 3px;
}
QStatusBar QLabel{
color: #E0E0E0;
padding: 0;
margin-left: 2px;
}

QTabWidget {
padding: 2px;
selection-background-color: #808080;
}
QTabWidget::pane {
border: 1px solid #606060;
border-radius: 3px;
padding: 2px;
margin: 0px;
}
QTabBar::tab:bottom {
background-color: #909090;
color: #606060;
margin-left: 2px;
padding-left: 4px;
padding-right: 4px;
padding-top: 2px;
padding-bottom: 2px;
min-width: 5px;
border-top: 3px solid #707070;
border-bottom-left-radius: 3px;
border-bottom-right-radius: 3px;
}
QTabBar::tab:bottom:selected {
color: #404040;
background-color: #C0C0C0;
border-top: 3px solid #606060;
border-bottom-left-radius: 3px;
border-bottom-right-radius: 3px;
}
QTabBar::tab:bottom:!selected:hover {
background-color: #A0A0A0;
color: #404040;
border: 1px solid #585858;
border-top: 3px solid #585858;
padding: 0px;
}

QGraphicView{
background-color: #606060;
}
QColumnView{
background-color: #606060;
}

/* QAbstractScrollArea ---------------------------------------------------- */

QAbstractScrollArea {
    background-color: #808080;
    border: 1px solid #606060;
    border-radius: 4px;
    padding: 4px;
    color: #F0F0F0;
}

QAbstractScrollArea:disabled {
    color: #787878;
}

/* QScrollArea ------------------------------------------------------------ */

QScrollArea QWidget{
background-color: #606060;
}

QScrollArea QWidget QWidget:disabled {
background-color: #808080;
}

/* QScrollBar ------------------------------------------------------------- */
QScrollBar:horizontal {
background-color: #909090;
height: 16px;
margin: 1px 16px 1px 16px;
border: 1px solid #707070;
border-radius: 2px;
}
/* scrollBar slider */
QScrollBar::handle:horizontal {
background-color: #C0C0C0;
border: 1px solid #808080;
min-width: 8px;
border-radius: 2px;
}
QScrollBar::handle:horizontal:hover {
background-color: #E0E0E0;
border: 1px solid #808080;
min-width: 8px;
border-radius: 2px;
}
/* page-down */
QScrollBar::add-line:horizontal {
margin: 0px 2px 0px 2px;
border-image: url(:/images/Resources/right_arrow_disabled.png);
width: 10px;
height: 10px;
subcontrol-position: right;
subcontrol-origin: margin;
}
QScrollBar::add-line:horizontal:hover,
QScrollBar::add-line:horizontal:on {
border-image: url(:/images/Resources/right_arrow.png);
height: 10px;
width: 10px;
subcontrol-position: right;
subcontrol-origin: margin;
}
/* page-up */
QScrollBar::sub-line:horizontal {
margin: 0px 2px 0px 2px;
border-image: url(:/images/Resources/left_arrow_disabled.png);
height: 10px;
width: 10px;
subcontrol-position: left;
subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal:hover,
QScrollBar::sub-line:horizontal:on {
border-image: url(:/images/Resources/left_arrow.png);
height: 10px;
width: 10px;
subcontrol-position: left;
subcontrol-origin: margin;
}
/* left-right-arrow */
QScrollBar::up-arrow:horizontal,
QScrollBar::down-arrow:horizontal {
background: none;
}
QScrollBar::add-page:horizontal,
QScrollBar::sub-page:horizontal {
background: none;
}

QScrollBar:vertical {
background-color: #909090;
width: 16px;
margin: 16px 1px 16px 1px;
border: 1px solid #707070;
border-radius: 2px;
}
/* scrollBar slider */
QScrollBar::handle:vertical {
background-color: #C0C0C0;
border: 1px solid #808080;
min-height: 8px;
border-radius: 2px;
}
QScrollBar::handle:vertical:hover {
background-color: #E0E0E0;
border: 1px solid #808080;
min-height: 8px;
border-radius: 2px;
}
/* page-down */
QScrollBar::add-line:vertical {
margin: 2px 0px 2px 0px;
border-image: url(:/images/Resources/down_arrow_disabled.png);
height: 10px;
width: 10px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover,
QScrollBar::add-line:vertical:on {
border-image: url(:/images/Resources/down_arrow.png);
height: 10px;
width: 10px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
/* page-up */
QScrollBar::sub-line:vertical {
margin: 2px 0px 2px 0px;
border-image: url(:/images/Resources/up_arrow_disabled.png);
height: 10px;
width: 10px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover,
QScrollBar::sub-line:vertical:on {
border-image: url(:/images/Resources/up_arrow.png);
height: 10px;
width: 10px;
subcontrol-position: top;
subcontrol-origin: margin;
}
/* up-down-arrow */
QScrollBar::up-arrow:vertical,
QScrollBar::down-arrow:vertical {
background: none;
}
QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
background: none;
}

QMessageBox {
background-color: #404040;
border: 2px solid #606060;
}
QMessageBox QLabel {
color: #C0C0C0;
}
QMessageBox QPushButton {
background-color: #606060;
color: #E0E0E0;
border: 1px solid #808080;
border-radius: 2px;
padding-top: 4px;
padding-bottom: 4 px;
padding-left: 20px;
padding-right: 20px;
margin-right: 12px;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>360</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_process">
       <attribute name="title">
        <string>Process</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="HistogramWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>180</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>180</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>340</width>
             <height>471</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_properties">
       <attribute name="title">
        <string>Properties</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QColumnView" name="columnView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="ImageScrollWidget" name="imageScrollWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="RenderWidget" name="openGLWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="actionZoom_In"/>
     <addaction name="actionZoom_Out"/>
     <addaction name="actionFit_To_Window"/>
     <addaction name="separator"/>
     <addaction name="actionZoom_25"/>
     <addaction name="actionZoom_50"/>
     <addaction name="actionZoomToNormal"/>
     <addaction name="actionZoom_200"/>
     <addaction name="actionZoom_400"/>
     <addaction name="actionZoom_800"/>
    </widget>
    <addaction name="menuZoom"/>
    <addaction name="actionCenter"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Exit"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionFit_To_Window"/>
   <addaction name="actionZoomToNormal"/>
   <addaction name="actionCenter"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Exit">
   <property name="icon">
    <iconset>
     <normalon>:/images/Resources/exit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset>
     <normalon>:/images/Resources/about.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normalon>:/images/Resources/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset>
     <normalon>:/images/Resources/zoomin.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset>
     <normalon>:/images/Resources/zoomout.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionFit_To_Window">
   <property name="icon">
    <iconset>
     <normalon>:/images/Resources/fittownd.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Fit To Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionZoomToNormal">
   <property name="icon">
    <iconset>
     <normalon>:/images/Resources/unzoom.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom 100%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="icon">
    <iconset>
     <normalon>:/images/Resources/center.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionZoom_25">
   <property name="text">
    <string>Zoom 25%</string>
   </property>
  </action>
  <action name="actionZoom_50">
   <property name="text">
    <string>Zoom 50%</string>
   </property>
  </action>
  <action name="actionZoom_200">
   <property name="text">
    <string>Zoom 200%</string>
   </property>
  </action>
  <action name="actionZoom_400">
   <property name="text">
    <string>Zoom 400%</string>
   </property>
  </action>
  <action name="actionZoom_800">
   <property name="text">
    <string>Zoom 800%</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>render_widget.h</header>
  </customwidget>
  <customwidget>
   <class>HistogramWidget</class>
   <extends>QWidget</extends>
   <header>histogram_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageScrollWidget</class>
   <extends>QWidget</extends>
   <header>image_scroll_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rawlab.qrc"/>
 </resources>
 <connections/>
</ui>
